aws/dual-region/scripts/generate_zeebe_helm_values.sh

```diff
@@ -52,23 +77,38 @@ if [[ "$namespace_0" == "$namespace_1" ]]; then
     exit 1
 fi
 
+if [ "$mode" = "failover" ] && [ "$lost_region" -ne 0 ] && [ "$lost_region" -ne 1 ]; then
+    echo "Invalid region $lost_region provided for the lost region. Please provide either 0 or 1 as input value."
+    exit 1
+fi
+
 # Generating and printing the string
 initial_contact=$(generate_initial_contact "$namespace_0" "$namespace_1" "$helm_release_name" "$cluster_size")
 elastic0=$(generate_exporter_elasticsearch_url "$namespace_0" "$helm_release_name")
 elastic1=$(generate_exporter_elasticsearch_url "$namespace_1" "$helm_release_name")
 
+if [[ "$mode" == "failover" ]]; then
+    if [[ "$lost_region" == 0 ]]; then
+        elastic0=$(generate_exporter_elasticsearch_url "$namespace_1_failover" "$helm_release_name")
+        elastic1=$(generate_exporter_elasticsearch_url "$namespace_1" "$helm_release_name")
+    else
+        elastic0=$(generate_exporter_elasticsearch_url "$namespace_0" "$helm_release_name")
+        elastic1=$(generate_exporter_elasticsearch_url "$namespace_0_failover" "$helm_release_name")
+    fi
+fi
+
 echo
-echo "Please use the following to change the existing environment variable ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS in the base Camunda Helm chart values file 'camunda-values.yml'. It's part of the 'zeebe.env' path."
+echo "Please use the following to change the existing environment variable ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS $target_text. It's part of the 'zeebe.env' path."
```

Comment:
in the failover mode, the user should change this in both the failover overlay and the base `camunda-values.yml`, currently only pointing to the overlay (same for the over env vars)

Reply:
added the base to the target text as well.